functions/Get-DbaSqlBuildReference.ps1

```diff
@@ -122,12 +146,35 @@ Function Get-DbaSqlBuildReference {
 			
 			$LastUpdated = Get-Date -Date $result.LastUpdated
 			if ($LastUpdated -lt (Get-Date).AddDays(-45)) {
-				Write-Message -Level Warning -Silent $Silent -Message "Index is stale, last update on: $(Get-Date -Date $LastUpdated -Format s)"
+				Write-Message -Level Warning -Silent $Silent -Message "Index is stale, last update on: $(Get-Date -Date $LastUpdated -Format s), try the -Update parameter to fetch the most up to date index"
 			}
 			
 			$result.Data | Select-Object @{ Name = "VersionObject"; Expression = { [version]$_.Version } }, *
 		}
 		
+		function Get-DbaSqlBuildReferenceIndexOnline {
+			[CmdletBinding()]
+			Param (
+				[bool]
+				$Silent
+			)
+			$url = 'https://sqlcollaborative.github.io/assets/dbatools-buildref-index.json'
```

Comment:
good thinking. But I don't want to disobey 'cause "disobedience shall be answered by the wrath of Fred." :-P I'm missing how to code the "handler" part for it.